src/tests/utilsTest.ts

```diff
@@ -8,80 +8,149 @@ import * as assert from "assert";
 import {utils as utils} from "../index"
 import {createDevOpsEnv} from "./helper"
 import * as fs from 'fs-extra'
+import {getTagsFromChangedFile,isTagExisting,getChangeFilesReadmeMap} from "../utils"
 
 
-@suite class UtilsTest {
-    @test async "TestGetOpenapiTypeDataplane" () {
-        let openapiType = await utils.getOpenapiType("./src/tests/Resource/openapi-type-data-plane-readme.md")
-        assert.equal(openapiType,"data-plane")
-    }
+@suite
+class UtilsTest {
 
-    @test async "TestGetOpenapiTypeDataplanArm" () {
-        let openapiType = await utils.getOpenapiType("./src/tests/Resource/openapi-type-arm-readme.md")
-        assert.equal(openapiType,"arm")
-    }
+  @test async "TestGetOpenapiTypeDataplane"() {
+    let openapiType = await utils.getOpenapiType("./src/tests/Resource/openapi-type-data-plane-readme.md")
+    assert.equal(openapiType, "data-plane")
+  }
 
-    @test async "TestGetOpenapiTypeNoExistFile" () {
-        let openapiType = await utils.getOpenapiType("C:/code/data-plane/test/readme.md")
-        assert.equal(openapiType,"default")
-    }
+  @test async "TestGetOpenapiTypeDataplanArm"() {
+    let openapiType = await utils.getOpenapiType("./src/tests/Resource/openapi-type-arm-readme.md")
+    assert.equal(openapiType, "arm")
+  }
 
-    @test async "TestGetOpenapiTypeFromPathWithArm" () {
-        let openapiType = await utils.getOpenapiType("C:/specification/test/resource-manager/test/readme.md")
-        assert.equal(openapiType,"arm")
-    }
+  @test async "TestGetOpenapiTypeNoExistFile"() {
+    let openapiType = await utils.getOpenapiType("C:/code/data-plane/test/readme.md")
+    assert.equal(openapiType, "default")
+  }
 
-    @test async "TestGetOpenapiTypeFromPathWithDataPlane" () {
-        let openapiType = await utils.getOpenapiType("/home/work/1/spec/specification/test/data-plane/test/readme.md")
-        assert.equal(openapiType,"data-plane")
-    }
+  @test async "TestGetOpenapiTypeFromPathWithArm"() {
+    let openapiType = await utils.getOpenapiType("C:/specification/test/resource-manager/test/readme.md")
+    assert.equal(openapiType, "arm")
+  }
 
-    @test async "TestGetOpenapiTypeFromRelativePath" () {
-        let openapiType = await utils.getOpenapiType("specification/test/data-plane/test/readme.md")
-        assert.equal(openapiType,"data-plane")
-    }
+  @test async "TestGetOpenapiTypeFromPathWithDataPlane"() {
+    let openapiType = await utils.getOpenapiType("/home/work/1/spec/specification/test/data-plane/test/readme.md")
```

Comment:
Why change the path?

Reply:
may the comparison problem, I haven't change it